test/helpers/helpers.go

```diff
@@ -235,3 +220,41 @@ func CheckCatalogSourceConfigAndChildResourcesDeleted(client test.FrameworkClien
 	}
 	return nil
 }
+
+// CheckSuccessfulCreationCsc checks that a CatalogSourceConfig
+// and it's child resources were deployed.
+func CheckSuccessfulCreationCsc(client test.FrameworkClient, cscName string, namespace string, targetNamespace string) error {
+	// Check that the CatalogSourceConfig was created.
+	resultCatalogSourceConfig := &marketplace.CatalogSourceConfig{}
+	err := WaitForResult(client, resultCatalogSourceConfig, namespace, cscName)
+	if err != nil {
+		return err
+	}
+
+	// Check that all child resources were created.
+	err = CheckCscChildResourcesCreated(client, cscName, namespace, targetNamespace)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// CheckSuccessfulDeletionCsc checks that a CatalogSourceConfig
+// and it's child resources were deleted.
+func CheckSuccessfulDeletionCsc(client test.FrameworkClient, cscName string, namespace string, targetNamespace string) error {
```

Comment:
Going with the naming pattern you are following this should be `CheckCscSuccessfulDeletion`